Makefile

```diff
@@ -1,6 +1,9 @@
 all: lkmpg.tex
 	pdflatex -shell-escap lkmpg.tex
 
+html:
+	pandoc -s lkmpg.tex -o lkmpg.html
```

Comment:
I checked the HTML file generated by `pandoc`, and It appears that HTML file is not recognizable for source code listing.

Related: https://github.com/sysprog21/lkmpg/pull/15

Reply:
It seems that `pandoc` did not include the `lib/codeblock.tex` properly.

Reply:
Since `make4ht` renders HTML better than `pandoc`, I would like to wait for #20 prior to this pull request.